crazyflie/scripts/crazyflie_server.py

```diff
@@ -767,6 +767,17 @@ def _init_parameters(self):
                                 value=cf_param_value,
                                 descriptor=parameter_descriptor,
                                 )
+                            # Based on the parameters from the last Crazyflie, set params for all
+                            # Warning: if any of the other crazyflies have different paramters    this will result in an error
```

Comment:
spelling:-)

Reply:
Thanks! btw, you can suggest a change in the line by using this button. Then I can commit directly here in the gui 😄 (i've just done that with my own comment)

![image](https://github.com/user-attachments/assets/a6fcb662-9327-4ca2-9c45-f92166d90531)